shinken/misc/common.py

```diff
@@ -80,7 +80,4 @@
 
 }
 
-try:
-    from setproctitle import setproctitle
-except ImportError as err:
-    setproctitle = lambda s: None
```

Comment:
I would raise  a proper Error exception instead of letting it crash. So that there is no "big" behavior difference